pyproject.toml

```diff
@@ -0,0 +1,7 @@
+[tool.black]
+# default is 88
+line-length = 120
+skip-string-normalization = true
```

Comment:
Oh good this is my primary complaint with black. We'll use pylint_quotes instead?

Reply:
Yep pylint_quotes will handle those issues